### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by DanielRosenwasser** [estree/estree#52]
An explicit license would be ideal to ensure that those writing tools around estree could do so without the risk of violating any copyright.

**Comment by sebmck** [comment]
Yep, I alluded to this in #44. The SpiderMonkey AST spec was published on MDN so it's *probably* under the license of [Creative Commons Attribution-ShareAlike license ](http://creativecommons.org/licenses/by-sa/2.5/) as specified in [About MDN](https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses). @dherman can possibly confirm or shed some light.

**Comment by sebmck** [comment]
@mikesherov Might be good to get a CLA too.

**Comment by mikesherov** [comment]
I'd prefer CC0, but CCAS is fine too. 

Re: CLA, I'm ambivalent to needing one, but if we think it's a good idea, I can see about getting one. ESLint and jQuery both have fine language in theirs that we can be inspired by.

**Comment by sebmck** [comment]
I can see it possibly being an issue for a large company that wanted to adopt it.

**Comment by mikesherov** [comment]
Ok, then I'll look into it.

**Comment by jmm** [comment]
This would be a good thing to get resolved. Aside from building tooling around it there's also prose involved, with no clear license to use it elsewhere and contributions from non-members being merged without any indication about what license they're providing.

**Comment by mikesherov** [comment]
Would anyone be opposed to slapping MIT on this?

**Comment by RReverser** [comment]
@mikesherov I'm fine with it. But I believe the most important is to get agreement from @dherman or others people who worked on SpiderMonkey AST in the first place, as we basically forked their spec.

**Comment by theduke** [comment]
Bump.

Need a license on this.
